news/2017-05-25-the-new-osutaiko-mascot-is-here.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "The new osu!taiko mascot is here!"
+permalink: http://osunews.tumblr.com/post/161051842473/the-new-osutaiko-mascot-is-here
+date: 2017-05-25 08:39:45 +0000
+tumblr_url: http://osunews.tumblr.com/post/161051842473/the-new-osutaiko-mascot-is-here
+---
+
+After almost a year in the making, the new osu!taiko mascot is finally ready to enter the scene. Say hello to **Mocha!**
+[[MORE]]
+
+<img src="https://assets.ppy.sh/media/mocha/mocha-header.jpg">
+
+Over ten months ago, [we held a contest](https://osu.ppy.sh/community/contests/2) to determine the visual design for the mascot of the osu!taiko gamemode. The contest descended into legend as we deliberated on the results - many of the community-made mascots rising into infamy through the various fan art contests we held in the meanwhile.
+
+It is time.
+
+We present to you, the brand new official mascot for osu!taiko - Mocha!<br><br><iframe src="https://docs.google.com/gview?url=https://assets.ppy.sh/media/mocha/Mocha_Documentation.pdf&embedded=true" style="width:750px; height:850px;" frameborder="0"></iframe>
```

Comment:
We've already discussed this in #osu-wiki iirc. News posts will be allowed HTML.